docs/glossary.md

```diff
@@ -5,35 +5,84 @@ sidebar_label: Glossary
+| Assessment Action             | Specific tasks or activities within the overall Assessment. These are the actual tests, evaluations, or analyses carried out - In Argus case the code that runs the compliance checks.                                                          |
+| Assessment Template           | Set of rules that can be applied to, and associated with multiple controls.                                                                                                                                                                     |
+| Assessor                      | Role that validates the Control Requirements.                                                                                                                                                                                                   |
+| Authorizing Official          | Someone with the authority to formally assume responsibility for operating an information system at an acceptable level of risk.                                                                                                                |
+| Baseline                      | See 'Control Baseline'.                                                                                                                                                                                                                         |
+| Catalog                       | See 'Control Catalog'.                                                                                                                                                                                                                          |
+| Catalog Model                 | 'A structured, machine-readable representation of a Catalog of Controls' (OSCAL Terminology).                                                                                                                                                   |
+| Compliance Framework Owner    | The application owner of Compliance Framework.                                                                                                                                                                                                  |
+| Control                       | Policies and procedures designed to ensure systems are secure and/or stable and/or resilient. (aka Requirement, or Guideline)                                                                                                                   |
+| Control Baseline              | 'A specific set of selected security control requirements from one or more control catalogs for use in managing risks in an information system' (OSCAL Terminology). Also known as an 'Overlay'.
+                                  'the set of controls that are applicable to information or an information system to meet legal, regulatory, or policy requirements, as well as address protection needs for the purpose of managing risk' (NIST 800-37)                         |
```

Comment:
Needs proper formatting.
Is this the definition of 'Overlay' or should it be incorporated with 'Control Baseline'?

Reply:
Yeah, they're the same @robel-yemane: https://pages.nist.gov/OSCAL/concepts/layer/#:~:text=Organizations%20and%20system%20owners%20identify%20which%20controls%20are%20applicable%20to%20a%20system%2C%20which%20may%20include%20controls%20from%20more%20than%20one%20catalog.%20These%20are%20often%20referred%20to%20as%20a%20baseline%20or%20overlay.